server/imports/api/lab.env.js

```diff
@@ -155,12 +180,11 @@ env.prototype.init = function(opts){
 	        reject(err);
 	      }
 	      else {
-               
                 //create redrouter etcd record
                 var etcd_redrouter = {
                   docker_container: containerId,
-                  port: 22,
-                  username: "root",
+                  port: slf.system.ssh_port,
```

Comment:
All of the code relating to constructing the etcd records needs to be inside the async functions